docs/root/configuration/access_log.rst

```diff
@@ -360,9 +360,57 @@ The following command operators are supported:
   TCP
     The subject present in the peer certificate used to establish the downstream TLS connection.
 
+%DOWNSTREAM_PEER_ISSUER%
```

Comment:
Should this be DOWNSTREAM_TLS_PEER_ISSUER to make it clear this refers to TLS.

Reply:
Good question, and one that I wrestled with as well. In a clean room, I would have preferred `DOWNSTREAM_TLS_PEER_ISSUER`; however, there was already some precedent set in https://github.com/envoyproxy/envoy/commit/65c05cad96db7f7874e1e5a2b70829dcfad10e88 (included in 1.10) for omitting the `TLS` prefix when referring to the peer and local certificates in access logs.

I opted for consistency with that precedent, but perhaps I should revisit? Let me know what you think.

Reply:
Ah, yeah. I guess consistency is probably best.